server/queries/pharamcies/eidtRequestsQuery.ts

```diff
@@ -0,0 +1,21 @@
+import { Product, ProductsRequest } from '../../models';
+
+const eidtRequestsQueryQuantity = (
+  idRequest: number,
+  idProduct: number,
+  quantity: number
+) =>
+  ProductsRequest.update(
+    { quantity },
+    {
+      where: { product_id: idProduct, id: idRequest, status: 'Pending' },
```

Comment:
`where: { id: requestId, status: 'Pending' }` is enough no need to check by both `product_id` & `id`